ts/util/guards.ts

```diff
@@ -63,6 +64,20 @@ export const is = <Ts extends Entity>(type: keyof TypeMap<Ts>) => {
   return nodeIs(typeMap)
 }
 
+/**
+ * A type guard to determine whether a node is of a particular type.
+ * Returns a boolean value and narrows the TypeScript inferred type to
+ * the type.
+ *
+ * @param type The type to test for
+ */
+export const isa = <K extends keyof Types>(
```

Comment:
What do you think about using camelCase to match the other functions, and changing to `isA`

```suggestion
export const isA = <K extends keyof Types>(
```